data/path.js

```diff
@@ -0,0 +1,9 @@
+import { isValidStringProperty } from "../validation/string";
+
+export function buildObjectsPath(id, className) {
+  let path = `/objects`;
+  if (isValidStringProperty(className)) {
+    path = `${path}/${className}`;
+  }
+  return `/${path}/${id}`;
```

Comment:
I  think that the first character: `/` can be omitted bc now you're always creating paths like that: `//objects/....` should be: `/objects/...`

Reply:
Ah yes, a typo. Thanks